src/handlers/createProduct/handler.test.ts

```diff
@@ -0,0 +1,91 @@
+import { createProduct } from './handler';
+import * as AWSMock from 'aws-sdk-mock';
+import * as AWS from 'aws-sdk';
+import { APIGatewayProxyEvent } from 'aws-lambda';
+import { IProduct } from '../../types/products';
+import { v4 as uuidv4 } from 'uuid';
+
+AWSMock.setSDKInstance(AWS);
+
+describe('createProduct', () => {
+  let event: APIGatewayProxyEvent;
+
+  beforeEach(() => {
+    const newProduct = {
+      title: "Test1",
+      description: "Test1 Description",
+      price: 100,
+      count: 1
+    };
+    event = {
```

Comment:
It's not necessary to describe the whole object, you can just describe needed fields

const event = { .... } as APIGatewayProxyEvent;

or

const event = { .... } as unknown as APIGatewayProxyEvent;